### DISCUSSION TITLE
No web access to device

### DISCUSSION THREAD

**Discussion by tgoschuetz** [RavenSystem/esp-homekit-devices#1652] [General]
I have a device configured and running, but cannot access the web interface via port 4567 after configuration. I have to set it back to setup mode in order to access the config page but then cannot reach it afterwards. Device answers on pings. Do you have any idea on how to dive into this? Everything was fine when I had the device running on my desk with UART connected to monitor but now I have it installed in my distribution cabinet where I cannot access the UART.

**Comment by ShareCat** [comment]
While you are at running mode, the web interface is disabled, but you can activate the setup mode(which will reboot first, then) enter into the web interface, check this out: https://github.com/RavenSystem/esp-homekit-devices/wiki/Setup-Mode#Entering-Setup-Mode
